pages.it/windows/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Mostra informazioni riguardo alla directory corrente e permette di muoversi in un altra directory.
+> Maggiori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/cd>.
+
+- Mostra il percorso(path) della directory corrente:
+
+`cd`
+
+- Spostati nella directory superiore:
+
+`cd ..`
+
+- Spostati ad una specifica directory sullo stesso disco:
+
+`cd {{percorso\alla\directory}}`
```

Comment:
```suggestion
`cd {{percorso\della\directory}}`
```

For paths in Italian, we are in the process of translating all instances of "al", "alla", "allo", etc. into "del", "della", "dello". We can use "della" here.

Reply:
> For paths in Italian, we are in the process of translating all instances of "al", "alla", "allo", etc. into "del", "della", "dello". We can use "della" here.


Thanks @mebeim , I will use this as a reference for future suggestions.